packages/adaptive_scaffold/README.md

```diff
@@ -1,18 +1,35 @@
 <?code-excerpt path-base="excerpts/packages/adaptive_scaffold_example"?>
 
-# Helper Widgets for Making Adaptive Layouts in Flutter (AdaptiveScaffold)
+# Helper Widgets for Making Adaptive Layouts in Flutter (`AdaptiveScaffold`)
 
-This package contains some helper widgets that make the process of developing adaptive layouts easier, especially with navigational elements.
+This package contains some helper widgets that make the process of developing
+adaptive layouts easier, especially with navigational elements.
 
-To see examples of using these helper widgets to make a simple but common adaptive layout:
+To see examples of using these helper widgets to make a simple but common
```

Comment:
Instead of `helper widgets`, let's just call them widgets.

Reply:
Done.